### PR TITLE
 net-im/telegram-desktop-bin: bump to 1.3.10 and disable auto-updates 

### PR DESCRIPTION
**Pull request by henning-schild**

### CODE REVIEW

**Comment by NP-Hardass** [comment]
@henning-schild if this closes out the bug, your tag in the first commit should be "Closes" not "Bugs"

How exactly was this issue resolved?  Do our users need to do anything specifically, or is it handled in the build automatically?

**Comment by henning-schild** [comment]
The new ebuild makes sure the software is always started with the updater disabled, users just have to upgrade to that new version. 

I think I have never seen Closes in that context, will check.

**Comment by henning-schild** [comment]
@NP-Hardass switched to using "Closes"

File: net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	>=sys-apps/dbus-1.4.20
+	x11-libs/libX11
+	>=x11-libs/libxcb-1.10[xkb]
+"
+DEPEND=""
+
+S="${WORKDIR}/Telegram"
+
+src_install() {
+	into /usr/lib/${PN}/
```
Comment (Zlogene) [inline net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild]: not that you need ending slash here

File: net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	newbin "${S}/Telegram" ${PN}
+	into /usr/
```
Comment (Zlogene) [inline net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild]: the same

File: net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+QA_PREBUILT="usr/lib/${PN}/bin/${PN}"
+
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/gobject-introspection
+	>=sys-apps/dbus-1.4.20
+	x11-libs/libX11
+	>=x11-libs/libxcb-1.10[xkb]
+"
+DEPEND=""
```
Comment (Zlogene) [inline net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild]: one more nitpick: why do you need empty variable then?

Comment (henning-schild) [reply net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild]: good point! was inherited from a previous ebuild, included a commit that drops that from all ebuilds

Comment (Zlogene) [reply net-im/telegram-desktop-bin/telegram-desktop-bin-1.3.10.ebuild]: @henning-schild do not forget to squash it as well

**Pull request closed without merging**